Find a functionally equivalent code:android.net.wifi.WifiInfo.writeToParcel:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Implement the Parcelable interface {@hide}
 */

@@ -2,7 +2,8 @@
     dest.writeInt(mNetworkId);
     dest.writeInt(mRssi);
     dest.writeInt(mLinkSpeed);
+    dest.writeInt(mFrequency);
     if (mIpAddress != null) {
         dest.writeByte((byte) 1);
         dest.writeByteArray(mIpAddress.getAddress());
@@ -17,21 +18,29 @@
     dest.writeString(mBSSID);
     dest.writeString(mMacAddress);
     dest.writeInt(mMeteredHint ? 1 : 0);
+    dest.writeInt(score);
+    dest.writeDouble(txSuccessRate);
+    dest.writeDouble(txRetriesRate);
+    dest.writeDouble(txBadRate);
+    dest.writeDouble(rxSuccessRate);
+    dest.writeInt(badRssiCount);
+    dest.writeInt(lowRssiCount);
     mSupplicantState.writeToParcel(dest, flags);
 }

